src/mgmt/cli_pool.c

```diff
@@ -37,6 +37,14 @@ struct pool_create_arg {
 	d_rank_list_t		*svc;
 };
 
+struct mgmt_list_pools_arg {
+	struct dc_mgmt_sys     *sys;
+	crt_rpc_t	       *rpc;
+	daos_mgmt_pool_info_t  *pools;
+	daos_size_t		req_npools;
+	daos_size_t	       *npools;
+};
+
```

Comment:
Might as well place this near mgmt_list_pools_cp.

Reply:
Sure, will do